string FORWARD = FORWARD
string BACK = BACK
string LEFT = LEFT
string RIGHT = RIGHT
string INVALID = INVALID
string DIRECTION
